recipes/privacy_meter/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "privacy-meter" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/privacy_meter-{{ version }}.tar.gz
+  sha256: c24a25f33b7bb15bc042ba211640bab4ed7c6e1f3ba93475e572aceb0550b147
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
```

Comment:
It looks like the upstream package doesn't properly specify it's requirements and tests don't pick up the issue because of an empty top-level `__init__.py`. Please review the `requirements.txt` and see if those are actually needed to run the package.

Reply:
You can see that it's importing a bunch of outside dependencies here for example: https://github.com/privacytrustlab/ml_privacy_meter/blob/master/privacy_meter/audit_report.py#L7-L14